checker/build.gradle

```diff
@@ -375,3 +375,39 @@ task jtregJdk11Tests(dependsOn: ':downloadJtreg', group: 'Verification') {
 
     }
 }
+
+group = 'org.checkerframework'
+version = '3.1.2-SNAPSHOT'
+
+
+publishing {
+    publications {
+        shadow(MavenPublication) { publication ->
+            project.shadow.component(publication)
+            artifactId = 'checker'
+            pom {
+                name = 'My Library'
```

Comment:
Copy-pasted from Gradle documentation for now.  Ideally we'd pull this from elsewhere too.  But since this is only for a local install, the contents are not that important.